apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/VedtaksvurderingRepository.kt

```diff
@@ -340,6 +341,17 @@ class VedtaksvurderingRepository(
             ) { it.toUtbetalingsperiode() }
         }
 
+    fun hentAvkortetYtelsePerioder(
+        vedtakId: Long,
+        tx: TransactionalSession? = null,
+    ): List<AvkortetYtelsePeriode> =
+        tx.session {
+            hentListe(
+                queryString = "SELECT * FROM avkortet_ytelse_periode WHERE vedtakid = :vedtakid",
+                params = { mapOf("vedtakid" to vedtakId) },
+            ) { it.toAvkortetYtelsePeriode() }
+        }
+
```

Comment:
Ser ikke ut til at `kotliquery` støtter IN-clause? Hadde vært fordelaktig å kunne spørre for samtlige vedtak i en smell.

Reply:
```
    fun hentVedtaker(vedtakIder: List<Long>, tx: TransactionalSession? = null): List<Vedtak> =
        tx.session {
            hentListe(
                """
                    SELECT sakid, behandlingId, saksbehandlerId, beregningsresultat, avkorting, vilkaarsresultat, id, fnr, 
                        datoFattet, datoattestert, attestant, datoVirkFom, vedtakstatus, saktype, behandlingtype, 
                        attestertVedtakEnhet, fattetVedtakEnhet, type, revurderingsaarsak, revurderinginfo, opphoer_fom,
                        tilbakekreving, klage 
                    FROM vedtak 
                    WHERE id = ANY (:vedtakIder)
                """.trimIndent(),
                {mapOf("vedtakIder" to
                        this.connection.underlying.createArrayOf("bigint", vedtakIder.toTypedArray()))}
            ) {
                val utbetalingsperioder = hentUtbetalingsPerioder(it.long("id"), this)
                it.toVedtak(utbetalingsperioder)
            }
        }
```

ser ut til å fungere for å hente flere med id i kotliquery